python/lsst/ts/eas/weather_model.py

```diff
@@ -29,12 +29,26 @@
 
 import astropy.units as u
 import lsst_efd_client
+import yaml
 from astropy.time import Time
 from lsst.ts import salobj, utils
 
 from .diurnal_timer import DiurnalTimer
 
 SAL_TIMEOUT = 60  # SAL timeout time. (seconds)
+TEMPERATURE_CUTOFF_TIME = (
+    5 * 60
+)  # Maximum age of temperature data to continue. (seconds)
+MAX_TEMPERATURE_SAMPLES = 20  # The maximum number of temperature samples to retain.
+
+NIGHT_SEARCH_WINDOW = 18 * u.hour
+DAY_SEARCH_WINDOW = 25 * u.hour
+
+SECONDS_PER_DAY = 86400
```

Comment:
you define the other constants with their unit and this one not. Later, on line 257 which is the only use of this constant, you apply the unit. Wouldn't it be easier to apply that here already?